options.rb

```diff
@@ -93,6 +109,26 @@ def list_authors
     end
   end
 
+  def list_labels
+    if @labels.empty?
+      puts 'No labels available.'
+    else
+      puts 'List of Labels:'
+      @labels.each do |label|
+        puts "ID: #{label.id}, Title: #{label.title}, Color: #{label.color}"
+        puts 'Items:'
+        if label.items.empty?
+          puts 'No items associated with this label.'
+        else
+          label.items.each do |item|
+            puts "- #{item.title}"
```

Comment:
There seems to be an issue when listing the labels. An error pops up saying that item.title is not a method. This is probably because inside of @labels you have items, and items don't have a title property. apart from that, the label list seems to be working correctly. Please check out the issue and make sure that the label list works as intended 😊